src/components/molecules/ZopaFooter/__snapshots__/ZopaFooter.test.tsx.snap

```diff
@@ -219,6 +219,7 @@ exports[`<ZopaFooter /> renders the component with no a11y violations 1`] = `
   font-weight: 400;
   -webkit-text-decoration: none;
   text-decoration: none;
+  color: #3B46C4;
```

Comment:
isn't a change, it looked like this before just wasn't specified (the links being purple on footer)